webapp/src/main/webapp/templates/freemarker/body/siteAdmin/siteAdmin-siteStyle.ftl

```diff
@@ -0,0 +1,72 @@
+<div>
+    <h2>Site Styles</h2>
+
+
+   <h3>Custom CSS Style</h3>
+
+   <#if customCssPath?? && customCssPath != "null">
+       <button
+       style="font-weight: normal !important; color: white !important; background-color: #b40000 !important; font-size: 16px !important; display: inline !important; visibility: visible !important; opacity: 1 !important; z-index: 10 !important; width: 201px !important; height: 31px !important;"
+        id="resetAction" type="button" class="button red" onclick="resetStyles()">Remove Custom CSS</button>
+       <a href="${customCssPath}" download="custom.css" class="button blue">Download Custom CSS</a>
+   </#if>
+   <form action="${actionUpload}" method="post" enctype="multipart/form-data">
+       <label for="fileUpload" class="">Upload new custom style.css:</label>
```

Comment:
the file doesn't need to be named style.css so maybe change this text to "Upload new css file" ?